CHANGELOG.rst

```diff
@@ -28,10 +28,28 @@ Changes are ordered reverse-chronologically.
  - KOLIBRI_LISTEN_PORT environment variable for specifying a default for the --port option #1724
 
 
+0.4.6
+-----
+
+ - Fix various exam and progress tracking issues
+ - Add automatic sign-out when browser is closed
+ - Fix search issue
+ - Learner UI updates
```

Comment:
Maybe add 'updated hindi translations' here? We can cherry-pick the commit back to the 0.4 branch

Reply:
OK, can do. Will need to update translations on 0.5 as well.

Reply:
Will do in the same PR.